openrtb/video.go

```diff
@@ -4,8 +4,8 @@ package openrtb
 //easyjson:json
 type Video struct {
 	MimeTypes   []string `json:"mimes"`
-	MinDuration int      `json:"minduration"`
-	MaxDuration int      `json:"maxduration"`
+	MinDuration *int     `json:"minduration,omitempty"`
```

Comment:
@garukun. Currently, openrtb.BidRequest only validate openrtb.Application. We may need to create another pull request to deal with the BidRequest validation.